updatecli/scripts/check-jdk.sh

```diff
@@ -52,7 +52,8 @@ case "${1}" in
   19.*+*)
     platforms=("x64_linux" "x64_windows" "aarch64_linux" "s390x_linux");;
   21*+*)
-    platforms=("x64_linux" "x64_windows" "aarch64_linux" "s390x_linux");;
+    # No s390x support for JDK21 yet
```

Comment:
```suggestion
    # No s390x support for JDK21
```
According to https://github.com/adoptium/temurin/issues/8 there won't be any s390x support for JDK21.

Reply:
Quoting Tim Ellison from Adoptium [in their Slack](https://adoptium.slack.com/archives/C09NLQQAV/p1697121672337069?thread_ts=1697112748.718609&cid=C09NLQQAV):

> As George and Stewart said above, the reason there is no Temurin 21 for ppc64 AIX/s390x Linux/arm32 Linux at the moment is because the OpenJDK 21 GA implementation on those platforms is not passing our release tests. When the fixes are available in OpenJDK source then we can proceed with a release.

So we may see s390x JDK21 builds in the near future (with 21.0.1 for example).
We won't see s390x JDK21+35 builds though.